api/google-search/src/main/java/com/google/search/googlesearch/services/SearchAutoCompleteService.java

```diff
@@ -0,0 +1,71 @@
+package com.google.search.googlesearch.services;
+
+import java.util.ArrayList;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class SearchAutoCompleteService {
+
+	private ArrayList<String> keywordArray;
+
+	public ArrayList<String> matchedKeywords;
+
+
+	public SearchAutoCompleteService() {
+		super();
+		this.getKeywordArrayList();
+	}
+
+	public ArrayList<String> findBySearchKeyword(String keyword) {
+		matchedKeywords = new ArrayList<String>();
+		for (String word : keywordArray) {
```

Comment:
Try to use JAVA8 wherever possible.

Reply:
Have added new function with `Stream` and `Lambda API` for filtering and collecting the result.